docs/configuration/configuration.md

```diff
@@ -98,7 +98,23 @@ remote_read:
 
 # Storage related settings that are runtime reloadable.
 storage:
-  [ - <exemplars> ... ]  
+  [ - <exemplars> ... ]
+
+# Tracing configures exporting traces from Prometheus to a tracing
+# backend via the OTLP protocol. Tracing is currently an experimental
+# feature and could change in the future.
+tracing:
+  # Client used to export the traces. Options are 'http' or 'grpc'.
+  client_type: <string>
+
+  # Endpoint to send the traces to.
+  endpoint: <string>
+
+  # Sets the probability a given trace will be sampled. Must be a float from 0 through 1.
+  [ sampling_fraction: <float> | default = 0 ]
+
+  # If enabled, the client will use a secure connection.
+  [ with_secure: <bool> | default = false ]
```

Comment:
Sorry, my request was unclear here. What I meant was a stub like this:

```yaml
tracing:
  [ <tracing> ]
```

I think the config should still have a separate section like before.

Reply:
Fixed, thanks :raised_hands: